CHANGELOG.md

```diff
@@ -14,24 +14,7 @@
 <!--    ### Improvements                                                -->
 <!--    ### Fixes                                                       -->
 
-# 5 April 2021
-## nist800-53-r5 v1.3
+# 18 May 2021
 ### Fixes
-- Fixed a bug where subcontrol-of relationships were not being created between controls and enhancements in nist-800-53-r5. See issue [#61](https://github.com/center-for-threat-informed-defense/attack-control-framework-mappings/issues/61).
-- Updates to mappings in nist-800-53-r5 to address withdrawn controls (SA family) and to remove policy control (XX-1) mappings.
-
-# 3 February 2021
-## nist800-53-r4 v1.2 and nist800-53-r5 v1.2
-### Fixes
-- Fixes parse_mappings.py for nist800-53-r4 and nist800-53-r5 to remove duplicate mappings. See issue [#58](https://github.com/center-for-threat-informed-defense/attack-control-framework-mappings/issues/58).
-
-# 7 January 2021
-## nist800-53-r4 v1.1 and nist800-53-r5 v1.1
-### Fixes
-- Fixed broken regex which was leading to erroneous mappings, in particular mappings to control enhancements. See issue [#56](https://github.com/center-for-threat-informed-defense/attack-control-framework-mappings/issues/56).
-
-# 15 December 2020
-## nist800-53-r4 v1.0 and nist800-53-r5 v1.0
-- Initial release of security control framework mapping methodology and tools. 
-- Initial release of NIST 800-53 R4 mappings to ATT&CK version 8.1.
-- Initial release of NIST 800-53 R5 mappings to ATT&CK version 8.1.
\ No newline at end of file
+- NIST 800-53 R4 layers updated with attack-navigator 4.3.
+- NIST 800-53 R5 layers updated with attack-navigator 4.3.
```

Comment:
Updates to the changelog should not overwrite the previous contents of the changelog, rather the new update should be prepended to the changelog.

Reply:
Good points, I reverted the CHANGELOG and updated the mappingsToHeatmaps.py script values